src/design/plone/theme/browser/tile_collection_renderers.py

```diff
@@ -84,6 +84,17 @@ def getCommentsCount(self, item):
             'comments': conversation.total_comments()
         }
 
+    def is_external_link(self, item):
+        # accepts a Link object
+        obj = item.getObject()
+        remoteUrl = getattr(obj, 'remoteUrl', None)
+
+        if remoteUrl:
+            # so it's a Link
+            return not remoteUrl.startswith('${portal_url}')
```

Comment:
Credo sia ammesso anche ``${navigation_root_url}`` come parametro invece di ``${portal_url}``, in tal caso sarebbe anche quello da controllare.

Reply:
A livello redazionale ho visto che ritorna quello, come potrebbe arrivare navigation_root_url?

Reply:
Ho visto come può essere fatto, ho quindi gestito entrambi i casi